helpers_test.go

```diff
@@ -111,3 +111,19 @@ func TestValidateTargetTable(t *testing.T) {
 		require.Equal(t, "t_2_1__1", validTable.Name())
 	})
 }
+
```

Comment:
Added a test for statement that was panicking to make sure the problem was solved